runtime/crab/src/lib.rs

```diff
@@ -144,7 +144,6 @@ frame_support::construct_runtime! {
 		Scheduler: pallet_scheduler = 28,
 		Preimage: pallet_preimage = 29,
 		Proxy: pallet_proxy = 30,
-		Multisig: pallet_multisig = 31,
```

Comment:
Do we need to update the pallet prefix after 31?

Reply:
I suggest updating this in the finaln release.

Reply:
Please do it right now or create an issue.

Reply:
There is already an issue tracking this.